test/nodes/test_ranker.py

```diff
@@ -109,6 +109,41 @@ def test_ranker_get_batches():
     assert next(batches) == (all_queries[0:1], all_docs[0:1])
 
 
+@pytest.mark.unit
+def test_add_meta_fields_to_docs():
+    docs = [
+        Document(
+            content="dummy doc 1",
+            meta={
+                "str_field": "test1",
+                "empty_str_field": "",
+                "numeric_field": 2.0,
+                "list_field": ["item0.1", "item0.2"],
+                "empty_list_field": [],
+            },
+        ),
+        Document(
+            content="dummy doc 2",
+            meta={
+                "str_field": "test2",
+                "empty_str_field": "",
+                "numeric_field": 5.0,
+                "list_field": ["item1.1", "item1.2"],
+                "empty_list_field": [],
```

Comment:
Hey @vblagoje I'm testing the empty list here already. Do you think this is enough?

Reply:
Aha, cool

Reply:
I'm sorry; I missed it. However, does it make sense to test an empty list of meta fields, i.e:
```
docs_with_meta = ranker._add_meta_fields_to_docs(documents=docs,embed_meta_fields=[])
```     
Just that we don't break that contract by accident.

Reply:
Sounds good! I'll add that edge case as well.